Sources/TensorFlow/Optimizer.swift

```diff
@@ -355,3 +355,98 @@ public class AdaGrad<Model: Layer>: Optimizer
         update(&model.allDifferentiableVariables, along: direction)
     }
 }
+
+/// Adadelta Optimizer
```

Comment:
- Use the canonical name (the name in the paper) to describe this class. 
- End all comments with a period.
- Make the "O" in "Optimizer" be lowercased, for consistency with API documentation on other optimizers in this file.
```suggestion
/// ADADELTA optimizer.
```